src/components/PreferencesBodyApplication.vue

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+import Vue from 'vue';
+import Tabbed from '@/components/Tabbed/index.vue';
+import Tab from '@/components/Tabbed/Tab.vue';
+import PreferencesApplicationBehavior from '@/components/PreferencesApplicationBehavior.vue';
+import PreferencesApplicationEnvironment from '@/components/PreferencesApplicationEnvironment.vue';
+
+export default Vue.extend({
+  name:       'preferences-body-application',
+  components: {
+    Tabbed, Tab, PreferencesApplicationBehavior, PreferencesApplicationEnvironment
+  },
+  data() {
+    return { activeTab: 'environment' };
+  },
+  methods:    {
+    tabSelected({ tab }: { tab: any }) {
+      this.activeTab = tab.name;
+    }
+  }
+});
+</script>
+
+<template>
+  <tabbed
+    v-bind="$attrs"
+    default-tab="pull"
+    class="action-tabs"
+    :no-content="true"
+    @changed="tabSelected"
+  >
+    <tab
+      label="Environment"
+      name="environment"
+      :weight="0"
+    />
+    <tab
+      label="Behavior"
+      name="behavior"
+      :weight="1"
+    />
+    <div class="application-content">
+      <component :is="`preferences-application-${activeTab}`" />
+    </div>
+  </tabbed>
+</template>
+
+<style lang="scss" scoped>
+  .application-content {
+    padding: 0.75rem;
+  }
+
+  .action-tabs::v-deep li.tab {
+    margin-right: 0;
+    padding-right: 0;
+    border-bottom: 1px solid;
+    border-color: var(--border);
```

Comment:
Because this looks visually the same as coming from `.tabs`, we can only apply the border to `.active` (via `border-bottom: 1px solid var(--primary);` instead.

Reply:
I don't think that's true because the bottom border for each will be overlayed by the border for the parent tab. It's also possible I'm misinterpreting the message 😅